src/ip/tools.cc

```diff
@@ -78,7 +78,7 @@ Ip::ProbeTransport()
 #else
     debugs(3, 2, "IPv6 transport " << (EnableIpv6?"Available":"Disabled"));
     if (EnableIpv6 != IPV6_OFF) {
-        debugs(3, DBG_CRITICAL, "WARNING: BCP 177 violation. IPv6 transport forced OFF by build parameters.");
+        debugs(3, Critical(67), "WARNING: BCP 177 violation. IPv6 transport forced OFF by build parameters.");
```

Comment:
> PR description: Restricting "BCP 177 violation" WARNINGs does not work because the warnings are printed before Squid configuration is parsed. Future initialization improvements may fix this.

We do not have to give these messages IDs because controlling them does not work "as is" right now. If you want me to undo these two "BCP 177" changes, please let me know.

Reply:
This is fine with me. The probes timing may change as the configuration actions are refactored, so it is good to be ready for that here.